server.js

```diff
@@ -64,29 +66,49 @@ app.post('/', function(req, res) {
       return res.end();
     }
 
-    var reviewers = mentionBot.guessOwnersForPullRequest(
-      data.repository.html_url, // 'https://github.com/fbsamples/bot-testing'
-      data.pull_request.number, // 23
-      data.pull_request.user.login // 'mention-bot'
-    );
-
-    console.log(data.pull_request.html_url, reviewers);
+    // request config from repo
+    github.repos.getContent({
+      user: data.repository.owner.login,
+      repo: data.repository.name,
+      path: CONFIG_PATH,
+      headers: {
+        Accept: "application/vnd.github.v3.raw"
+      }
+    }, function(err, configRes) {
+      // default config
+      var repoConfig = {
+        userBlacklist: []
+      };
+
+      if (!err && configRes) {
+        repoConfig = JSON.parse(configRes);
```

Comment:
JSON.parse throws an exception if the argument is not valid json, can you add a try catch around it so that it doesn't crash the entire node process for everyone?